src/controllers/hid/hidiooutputreport.cpp

```diff
@@ -121,10 +122,15 @@ bool HidIoOutputReport::sendCachedData(QMutex* pHidDeviceAndPollMutex,
             reinterpret_cast<const unsigned char*>(m_lastSentData.constData()),
             m_lastSentData.size());
     if (result == -1) {
-        qCWarning(logOutput) << "Unable to send data to device :"
-                             << mixxx::convertWCStringToQString(
-                                        hid_error(pHidDevice),
-                                        kMaxHidErrorMessageSize);
+        if (!m_hidWriteErrorLogged) {
+            qCWarning(logOutput) << "Unable to send data to device :"
+                                 << mixxx::convertWCStringToQString(
+                                            hid_error(pHidDevice),
+                                            kMaxHidErrorMessageSize);
+            m_hidWriteErrorLogged = true; // Don't log the same error again
```

Comment:
This will not wor as the comment describe. To ensure that only identical errors are inhibited, you must compare the error message string.